app/views/issues_rss/index.xml.builder

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+xml.instruct!
+xml.rss version: '2.0', 'xmlns:atom': 'http://w3.org/2005/Atom', 'xmlns:georss': 'http://www.georss.org/georss' do
+  xml.channel do
+    xml.title Settings::Instance.name
+    xml.tag! 'atom:link', rel: 'self', type: 'application/rss+xml', href: request.url
+    xml.link root_url
+    xml.description t('.feed_description', name: Settings::Instance.name)
```

Comment:
Da das ja der persönliche RSS Feed für einen Nutzer ist, sollten wir hier ggf. auch den konkreten Namen des Nutzers mit aufnehmen? Damit man sieht, das es nur für diesen Nutzer ist?

Reply:
War im alten nicht, da habe ich mir die Vorlage hergeholt. können wir aber gerne hinzufügen.

Reply:
Bisheriger Text lautet:
`Diese Daten umfassen alle Ihre Meldungen in Klarschiff.SN.`

Eigentlich steht da ja bereits drin, dass es nur die Meldungen für die Aktuelle Person sind. Nur eben ohne den Namen.

Reply:
Das ist dann okay